### PR TITLE
Add basic scenario for wicked

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Comment by asmorodskyi** [comment]
@okurz , @coolo  - please tell what you think . It much up to now but representing general idea how I see it. Other people also welcome to express their opinion.

File: data/wicked/get_interfaces.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (okurz) [inline data/wicked/get_interfaces.sh]: `#!/bin/bash -ex`

File: tests/console/wicked_basic.pm [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# Summary: Sanity checks of wicked
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils 'systemctl';
+
+sub run {
+    systemctl "stop wicked.service";
```
Comment (okurz) [inline tests/console/wicked_basic.pm]: better be consistent, e.g. '()' around arguments

File: tests/console/wicked_before_test.pm [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Do basic checks to make sure system is ready for weaked testing
```
Comment (okurz) [inline tests/console/wicked_before_test.pm]: s/weaked/wicked/, LOL :-)

File: tests/console/wicked_before_test.pm [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+use testapi;
+use utils 'systemctl';
+
+sub run {
+
+    select_console('root-console');
+    systemctl('is-active network');
+    systemctl('is-active wicked');
+    assert_script_run('[ -z "$(coredumpctl -1 --no-pager --no-legend)" ]');
+    wait_idle;
```
Comment (okurz) [inline tests/console/wicked_before_test.pm]: no

File: tests/console/wicked_before_test.pm [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    script_run("curl -L -v " . autoinst_url . "/data/get_interfaces.sh > get_interfaces.sh");
+    script_run("chmod +x ./get_interfaces.sh");
```
Comment (okurz) [inline tests/console/wicked_before_test.pm]: Both of these should work with `assert_script_run. And better use single ticks by default

File: tests/console/wicked_basic.pm [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    assert_script_run("! systemctl is-active wicked.service");
+    systemctl('is-active wickedd');
+    script_run("./get_interfaces.sh");
```
Comment (okurz) [inline tests/console/wicked_basic.pm]: Sure, looks fine but I don't see the benefit to put it into another script

Comment (coolo) [inline tests/console/wicked_basic.pm]: the scripts will get more complicated.

File: tests/console/wicked_before_test.pm (hunk 4 above)
Comment (coolo) [inline tests/console/wicked_before_test.pm]: I wonder why though

File: tests/console/wicked_basic.pm [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+use base "consoletest";
+use strict;
+use testapi;
+use utils 'systemctl';
+
+sub run {
+    systemctl('stop wicked.service');
+    assert_script_run('! systemctl is-active wicked.service');
+    systemctl('is-active wickedd');
+    assert_script_run('for dev in /sys/class/net/*; do grep "down" $dev/operstate || (echo "device $dev is not down" && exit 1) ; done');
```
Comment (okurz) [inline tests/console/wicked_basic.pm]:
```bash
shopt -s extglob
for dev in /sys/class/net/!(lo); do grep "down" $dev/operstate || (echo "device $dev is not down" && exit 1) ; done
```

**Comment by asmorodskyi** [comment]
proof run http://kimball.arch.suse.de/tests/651

**Comment by coolo** [comment]
http://kimball.arch.suse.de/tests/651#step/wicked_basic/3 is impressive :)

**Comment by asmorodskyi** [comment]
just double check that in code I am entering "! systemctl is-active wicked.service'" so is some kind of ninja key reincarnation ?

**Comment by okurz** [comment]
Now please make sure the scenario is triggered and run on o3 in the test development group first, monitor for some builds, see it passing, move to validation group, inform reviewers, extend to SLE (move test schedule part to `lib/main_common`, trigger on osd, then extend

**Pull request merged**